### PR TITLE
First_try

### PR DESCRIPTION
**Pull request by NazarSalaban**

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://127.0.0.1:3000',
```
Comment (ovartem) [inline playwright.config.js]: you can add base url here to avoid entering `https://www.saucedemo.com/` in every test

File: test_1.spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@playwright/test';
+
+test('Log In and Checking element', async ({ page }) => {
+
+  await page.goto('https://www.saucedemo.com/');
```
Comment (ovartem) [inline test_1.spec.js:4]: Let's use [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each-1) function to perform login

**Review by ovartem — CHANGES_REQUESTED** [review]
Really great job!
Check those small comments left

File: test_1.spec.js (hunk 2 above)
Comment (NazarSalaban) [reply test_1.spec.js:4]: Fixed

File: playwright.config.js (hunk 1 above)
Comment (NazarSalaban) [reply playwright.config.js]: Fixed

**Review by hlazkov — APPROVED** [review]
Good job here! Lets proceed with the next task!

**Pull request merged**